tools/sim/bridge.py

```diff
@@ -1,553 +1,24 @@
-#!/usr/bin/env python3
```

Comment:
If I rename this file `run_bridge.py`, the line-by-line changes would be easier to review. The reviewer can let me know if they prefer that. Then when we merge, we can rename to bridge.py.

Reply:
renamed for now. Let's close this thread once we rename to `bridge.py`

Reply:
Rmb to do this before merge.